VimeoNetworking/Sources/Models/VIMConnection.h

```diff
@@ -71,6 +71,10 @@ extern NSString *const __nonnull VIMConnectionNameNotifications;
 @property (nonatomic, strong, nullable) NSNumber *mainVideosCount;
 @property (nonatomic, strong, nullable) NSNumber *viewableVideosCount;
 
+#pragma mark - Notifications related only
```

Comment:
Subclassing was my initial approach but didn't spend time on the impact to VOD. After syncing up with @nicolelehrer, I'll make subclasses for notifications and VOD.